### ISSUE TITLE
Feature Request - Loader Config Folder

### ISSUE THREAD

**Issue by bartboy011** [rails/webpacker#171]
Hey all,

I'm working on using PostCSS with Vue-Loader to autoprefix the scoped styles in my Vue templates. The PostCSS loader for Webpack requires a config file, but currently the only way I've managed to to get Webpack to recognize it is to place the postcss.config.js file inside one my pack folders in app/javascript. Maybe there's a simpler way to manage this that I've missed, in which case an update to the documentation would suffice, but it seems a folder specifically for loader config files would be an awesome addition. Something along the lines of the Rails config/initializers folder, maybe?

Thanks for this awesome tool, Rails was beginning to seem like it was falling behind in Front-end Development but this has brought it right back up!

**Comment by ytbryan** [comment]
Do you have that repo on github?

**Comment by bartboy011** [comment]
In a private repo, unfortunately (work project).

**Comment by gauravtiwari** [comment]
@bartboy011 Great idea, but it seems that this feature is out-of-scope. Adding post-css is rather 
project specific requirement. Perhaps, you can make a gem that depends on webpacker to extend the feature (webpacker-postcss). Btw, did you checked https://github.com/michael-ciniawsky/postcss-load-config. It seems this works much like `.babelrc` for babel-loader.

**Comment by bartboy011** [comment]
I agree that using it exclusively for a PostCSS config is out-of-scope, my thinking was more to create one folder for the config files of any Webpack plugin/loader that requires a config file, not just PostCSS. However, I will say that I'm unfamiliar with any other loader that requires a config file like PostCSS does. Maybe I'm overestimating the importance of having a config folder for loaders.

Regardless, the PostCSS Load Config package is super helpful, thank you!

**Comment by gauravtiwari** [comment]
@bartboy011 I see, yeah I have been thinking to split loaders into a separate folder after #153 to make things modular and also upgrades easier, but not sure about config files. As you mentioned, not all loaders need that kinda of configuration. 

Cool 👍  it's more like .babelrc right?

Feel free to close this one if you are happy 😄
